src/ajax/xhr.js

```diff
@@ -15,7 +16,10 @@ var xhrSuccessStatus = {
 
 		// File protocol always yields status code 0, assume 200
 		0: 200
-	};
+	},
+	xhrSupported = jQuery.ajaxSettings.xhr();
+
+support.responseURL = !!xhrSupported && ( "responseURL" in xhrSupported );
```

Comment:
From all of our supported (on `master`) browsers, only IE doesn't support `responseUrl`. Since #4387, we identify IE directly & don't define support tests if only IE is affected.

Reply:
Okay! I was doing it for IE 11 & Edge < 14, removing